cluster-diagnosis/sysdumpcollector.py

```diff
@@ -253,7 +253,8 @@ def collect_cilium_secret(self):
         cmd = "kubectl get secret cilium-etcd-secrets -n {} -o json".format(
             namespace.cilium_ns)
         try:
-            output = json.loads(subprocess.check_output(cmd, shell=True))
+            output = json.loads(
+                subprocess.check_output(cmd, shell=True).decode("utf-8"))
```

Comment:
Does this work with < Python 3.5?

Reply:
> Does this work with < Python 3.5?

I've manually tested on 2.7.10, 3.4.9, 3.5.6 and 3.7.2 and it's working fine.